modules/swagger-jaxrs2/src/main/java/io/swagger/jaxrs2/OperationParser.java

```diff
@@ -422,35 +436,6 @@ public static void applyProduces(Produces classProduces, Produces methodProduces
         return Optional.empty();
     }
 
-    public static Optional<MediaType> getMediaType(MediaType mediaType, ExampleObject example) {
```

Comment:
remove related unused imports IOException and Json

Reply:
Removed!